phpBB/phpbb/notification/method/base.php

```diff
@@ -104,6 +134,55 @@ public function add_to_queue(\phpbb\notification\type\type_interface $notificati
 	}
 
 	/**
+	* {@inheritdoc}
+	*/
+	public function update_notification($notification, array $data, array $options)
```

Comment:
Is there any valid use case when you do not overwrite these functions in a notification method object?